### PR TITLE
[Scalar-Schema] Set Cosmos secondary index

### PR DESCRIPTION
**Pull request by yito88**
https://scalar-labs.atlassian.net/browse/DLT-8021

This PR is for Cosmos DB

For Cassandra: #140 
For DynamoDB: #142 

Please merge after #140

### CODE REVIEW

**Review by feeblefakie — APPROVED** [review]
LGTM!

File: tools/scalar-schema/java/src/CosmosTableMetadata.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
   private String id;
```
Comment (brfrn169) [inline tools/scalar-schema/java/src/CosmosTableMetadata.java:17]: Just out of my curiosity. Why do we have CosmosTableMetadata in this schema tool project?

**Review by brfrn169 — APPROVED** [review]
Left one question, but it's just out of my curiosity. I'll approve this.

File: tools/scalar-schema/java/src/CosmosTableMetadata.java (hunk 1 above)
Comment (yito88) [reply tools/scalar-schema/java/src/CosmosTableMetadata.java:17]: The class is used for metadata insertion.
Cosmos DB SDK serializes/deserializes the data according to this class's setter and getter.

This tool gives the instance to the Upsert API.
https://github.com/scalar-labs/scalardb/blob/0f9cd909fd94ac04e46575860d0110ba6d57dab7/tools/scalar-schema/src/scalar_schema/cosmos.clj#L96

**Pull request merged**